CHANGES.rst

```diff
@@ -1,7 +1,102 @@
-0.18 (unreleased)
------------------
+0.18 (Nov 4th, 2020)
+--------------------
 
-- Planned for June 2020
+- Released November 4, 2020
+- 9 contributors
+- 160 pull requests (not all listed below)
+
+**What's new**
+
+Gammapy v0.18 comes with many new features, further unified API, bug fixes
+and improved performance.
+
+For Gammapy v0.18 the handling of the instrumental background was refactored
+by introducing a ``FoVBackgroundModel`` which is specific to each ``MapDataset``.
+Instead of relying on the previous ``BackgroundModel``, which couples
+the map template and spectral parameters, the information is now split such that
+``MapDataset.background`` contains the map template and the ``FoVBackgroundModel``
+the corresponding parameters. The model corrected background can now be
```

Comment:
the information is now split such that ``MapDataset.background`` and ``FoVBackgroundModel`` contain the map template and the corresponding parameters, respectively.